datalad/support/annexrepo.py

```diff
@@ -152,6 +156,12 @@ def _run_annex_command(self, annex_cmd, git_options=[], annex_options=[],
         debug = ['--debug'] if lgr.getEffectiveLevel() <= logging.DEBUG else []
         backend = ['--backend=%s' % backend] if backend else []
 
+        git_options = git_options[:] if git_options else []
+        annex_options = annex_options[:] if annex_options else []
```

Comment:
ah -- `[:]` wasn't needed what I meant... anyways -- ignore ;) better be safe than sorry